contact-us/index.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: default
+title: Contact Us
```

Comment:
We should not have a page that just redirects to another page.

Reply:
Having `Contact Us` redirect to new page leads to inconsistency. Should the new page display data from `get help` page instead of having a link that redirects to `get help` page?